docs/sources/flow/reference/components/otelcol.receiver.otlp.md

```diff
@@ -0,0 +1,183 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/otelcol.receiver.otlp
+title: otelcol.receiver.otlp
+---
+
+# otelcol.receiver.otlp
```

Comment:
I want to be able to give an example pipeline at the bottom of the page, but I can't at the moment, not until there's at least one exporter component.